provider/pkg/gen/dotnet-templates/helm/ChartBase.cs

```diff
@@ -84,7 +84,7 @@ protected ChartBase(string releaseName, Union<ChartArgs, LocalChartArgs> args, C
                         fetchOptions.Version = cfg.Version;
                         Fetch(chartToFetch, fetchOptions);
                         // Sort the directories into alphabetical order, and choose the first
-                        var fetchedChart = chartDirectory.GetDirectories().OrderBy(x => x.Name).ToArray()[0];
+                        var fetchedChart = chartDirectory.GetDirectories().OrderBy(x => x.Name).First();
```

Comment:
We can just use Linq's `First()` to get the first item, instead of allocating an array to get it.